_docs/tasks/traffic-management/egress.md

```diff
@@ -91,27 +84,86 @@ HTTP or port `443` for HTTPS.
    kubectl exec -it $SOURCE_POD -c sleep bash
    ```
 
-2. Make a request to an external service using the `name` from the Service spec
-   above followed by the path to the desired API endpoint:
+2. Make a request to the external HTTP service:
+
+   ```bash
+   curl http://httpbin.org/headers
+   ```
+
+3. Make a request to the external HTTPS service.
+   External services of type HTTPS must be accessed over HTTP with the port specified in the request:
+
+   ```bash
+   curl http://www.google.com:443
+   ```
+
+### Setting route rules on an external service
+
+Similar to inter-cluster requests, Istio
+[routing rules]({{home}}/docs/concepts/traffic-management/rules-configuration.html)
+can also be set for external services that are accessed using egress rules.
+To illustrate we will use [istioctl]({{home}}/docs/reference/commands/istioctl.html)
+to set a timeout rule on calls to the httpbin.org service.
+
+1. From inside the pod being used as the test source, invoke the `/delay` endpoint of the httpbin.org external service:
+
+   ```bash
+   kubectl exec -it $SOURCE_POD -c sleep bash
+   time curl -o /dev/null -s -w "%{http_code}\n" http://httpbin.org/delay/5
+   ```
+
+   ```bash
+   200
+
+   real    0m5.024s
+   user    0m0.003s
+   sys     0m0.003s
+   ```
+
+   The request should return 200 (OK) in approximately 5 seconds.
+
+1. Exit the source pod and use `istioctl` to set a 3s timeout on calls to the httpbin.org external service:
 
    ```bash
-   curl http://externalbin/headers
+   cat <<EOF | istioctl create -f -
+   apiVersion: config.istio.io/v1alpha2
+   kind: RouteRule
+   metadata:
+     name: httpbin-timeout-rule
+   spec:
+     destination:
+       service: httpbin.org
+     http_req_timeout:
+       simple_timeout:
+         timeout: 3s
+   EOF
    ```
 
-3. For external services of type HTTPS, the port must be specified in the request.
-   App clients should make the request over HTTP since the Egress Envoy will initiate HTTPS 
-   with the external service:
+1. Wait a few seconds, then issue the _curl_ request again:
 
    ```bash
-   curl http://securegoogle:443
+   kubectl exec -it $SOURCE_POD -c sleep bash
+   time curl -o /dev/null -s -w "%{http_code}\n" http://httpbin.org/delay/5
```

Comment:
Make the delay larger so that people don't see the 3 vs 5 as noise. Like use httpbin.org/delay/7 for example.

Reply:
Using `time curl ...` as the command and printing the http response, so the difference is very clear in the output when you run it (esp the 504 vs 200 response). I think waiting 5 secs for the command to return is already annoying enough :-)